v0/destinations/one_signal/data/OneSignalIdentifyConfig.json

```
@@ -0,0 +1,46 @@
+[
+  {
+    "sourceKeys": "context.device.model",
+    "destKey": "device_model",
+    "required": false
+  },
+  {
+    "sourceKeys": "context.os.version",
+    "destKey": "device_os",
+    "required": false
+  },
+  { "sourceKeys": "context.locale", "destKey": "laguage", "required": false },
+  {
+    "sourceKeys": "createdAt",
+    "destKey": "created_at",
+    "sourceFromGenericMap": true,
+    "required": false
+  },
+  {
+    "sourceKeys": "createdAt",
+    "destKey": "last_active",
+    "sourceFromGenericMap": true,
+    "required": false
+  },
+  {
+    "sourceKeys": [
+      "traits.country",
+      "context.traits.country",
+      "traits.address.country",
+      "context.traits.address.country"
+    ],
+    "destKey": "country",
+    "required": false
+  },
+  {
+    "sourceKeys": "context.timezone",
+    "destKey": "timezone",
+    "required": false
+  },
+  {
+    "sourceKeys": "userId",
```

```suggestion
    "sourceKeys": "userIdOnly",
```
As we are already setting the anonymousId as a tag

---

Updated

---

Updated